examples/stack/link_layer/example_link_layer_ck.py

```diff
@@ -45,9 +43,8 @@ def post_create(conn, q, pair):
             q.measure(array_entry)
 
         # Create EPR pair
-        epr_socket.create(
```

Comment:
In this file I am also not sure why the `epr_socket` is being kept.

Reply:
`create_keep` does not mean that the `epr_socket` is kept, but rather that the EPR pairs (entangled pairs) that are produced are kept in memory. This is in contrast to `create_measure` where the EPR pairs are directly measured, and hence not 'kept'.